plugins/catalog/src/components/CatalogPage/CatalogPage.tsx

```diff
@@ -52,9 +57,42 @@ const CatalogPageContents = () => {
   const userId = useApi(identityApiRef).getUserId();
   const [selectedTab, setSelectedTab] = useState<string>();
   const [selectedSidebarItem, setSelectedSidebarItem] = useState<string>();
+  const [entitiesState, setEntitiesState] = useState<Entity[]>([]);
+  const [errorState, setError] = useState<Error | undefined>();
+
+  useEffect(() => {
+    setError(error);
+    setEntitiesState(matchingEntities);
+  }, [error, matchingEntities]);
+  const catalogApi = useApi(catalogApiRef);
   const orgName =
     useApi(configApiRef).getOptionalString('organization.name') ?? 'Company';
 
+  const addMockData = async () => {
+    try {
+      const dummyEntities = [
+        'artist-lookup-component.yaml',
+        'playback-order-component.yaml',
+        'podcast-api-component.yaml',
+        'queue-proxy-component.yaml',
+        'searcher-component.yaml',
+        'playback-lib-component.yaml',
+        'www-artist-component.yaml',
+        'shuffle-api-component.yaml',
+      ];
+      const _promises = dummyEntities.map(file =>
+        catalogApi.addLocation(
+          'github',
+          `https://github.com/spotify/backstage/blob/master/packages/catalog-model/examples/${file}`,
```

Comment:
Similarly, this could go in config. This URL will likely change in the future if the donation of Backstage to the CNCF goes through.